### ISSUE TITLE
Integrate brakepad crash report library for core dumps

### ISSUE THREAD

**Issue by clemahieu** [clemahieu/raiblocks#461]

**Comment by androm3da** [comment]
I hadn't heard of [breakpad](https://chromium.googlesource.com/breakpad/breakpad/), sounds interesting.

If I understand the "[getting started](https://chromium.googlesource.com/breakpad/breakpad/+/master/docs/getting_started_with_breakpad.md)" page correctly it sounds as if the minidump could contain secrets that shouldn't leave the user's system.  Even if the user manually submits this to the dev team, how can they have confidence their keys aren't in there?

I'm assuming a minidump is similar in concept to a core file in that it could have much or all of the process' (sparse) virtual memory.  I can't imagine a reliable way to censor the secrets unless we read them from disk every time we need them.  This would narrow the risk significantly, though it would still be there.

**Comment by clemahieu** [comment]
True, I guess the highest priority thing is getting a stack trace that we can diagnose on, preferably with line numbers.  Whatever is the easiest/portable means to do that is really what we want.

**Comment by lukealonso** [comment]
Is the automatic reporting aspect important, or is just printing out a stack trace on crashes that people can provide manually good enough?

**Comment by lukealonso** [comment]
(we could write them to the log, for instance)
